### ISSUE TITLE
Python 3.13 removed 'pipes' module

### ISSUE THREAD

**Issue by 0-wiz-0** [jupyter/nbclassic#308]
When building nbclassic with Python 3.13, the build fails due to its use of the pipes library.
```
* Building wheel...
Traceback (most recent call last):
  File "/usr/pkg/lib/python3.13/site-packages/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
    main()
    ~~~~^^
  File "/usr/pkg/lib/python3.13/site-packages/pyproject_hooks/_in_process/_in_process.py", line 335, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
                             ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/pkg/lib/python3.13/site-packages/pyproject_hooks/_in_process/_in_process.py", line 251, in build_wheel
    return _build_backend().build_wheel(wheel_directory, config_settings,
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                        metadata_directory)
                                        ^^^^^^^^^^^^^^^^^^^
  File "/usr/pkg/lib/python3.13/site-packages/jupyter_packaging/build_api.py", line 23, in build_wheel
    val = orig_build_wheel(
        wheel_directory,
        config_settings=config_settings,
        metadata_directory=metadata_directory,
    )
  File "/usr/pkg/lib/python3.13/site-packages/setuptools/build_meta.py", line 421, in build_wheel
    return self._build_with_temp_dir(
           ~~~~~~~~~~~~~~~~~~~~~~~~~^
        ['bdist_wheel'],
        ^^^^^^^^^^^^^^^^
    ...<3 lines>...
        self._arbitrary_args(config_settings),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/pkg/lib/python3.13/site-packages/setuptools/build_meta.py", line 403, in _build_with_temp_dir
    self.run_setup()
    ~~~~~~~~~~~~~~^^
  File "/usr/pkg/lib/python3.13/site-packages/setuptools/build_meta.py", line 318, in run_setup
    exec(code, locals())
    ~~~~^^^^^^^^^^^^^^^^
  File "<string>", line 52, in <module>
  File "/tmp/www/py-nbclassic/work/nbclassic-1.1.0/setupbase.py", line 15, in <module>
    import pipes
ModuleNotFoundError: No module named 'pipes'

ERROR Backend subprocess exited when trying to invoke build_wheel
```
pipes was obsoleted in Python 3 (yes, 3.0) and finally removed in 3.13.
You can probably use shlex instead.

**Comment by 0-wiz-0** [comment]
That's great, thank you! Looking forward to the release.

**Comment by arteymix** [comment]
Is there a reason this fix was not included in 3.3.7? Clobber is currently broken in Fedora 41. https://bugzilla.redhat.com/show_bug.cgi?id=2335620
